style.css

```diff
@@ -198,6 +198,183 @@ body {
   z-index: 2;
 }
 
+.order {
+  background: #FAE8E4;
```

Comment:
Codacy found an issue: [Expected "#FAE8E4" to be "#fae8e4" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/LOCO_proj_Boiko/pullRequest?prid=5841754)